contributers/contributer.json

```diff
@@ -188,4 +188,9 @@
         "college": "CEG"
 
     }
```

Comment:
Shweta, Make sure you write in the same format required and update the PR accordingly.